amd_openvx_extensions/amd_rpp/include/internal_publishKernels.h

```diff
@@ -210,5 +211,6 @@ vx_status Nop_Register(vx_context);
 #define VX_KERNEL_RPP_COPY_NAME                                 "org.rpp.Copy"
 #define VX_KERNEL_RPP_CROP_MIRROR_NORMALIZE_NAME                "org.rpp.CropMirrorNormalize"
 #define VX_KERNEL_RPP_NOP_NAME                                  "org.rpp.Nop"
+#define VX_KERNEL_RPP_DOWNMIX_NAME "org.rpp.Downmix"
```

Comment:
Please align the "org.rpp.Downmix with previous declarations alignment

Reply:
Alignment Done